aliases/available/thefuck.aliases.bash

```diff
@@ -0,0 +1,5 @@
+# https://github.com/nvbn/thefuck
+alias fuck='eval $(thefuck $(fc -ln -1))'
+alias please='fuck'
```

Comment:
Let's be consistent in our use of quotes for the secondary aliases.

I know the original alias was quoted (i.e `please='fuck'`) but do the quotes here bring any value?

* If so, then let's quote all of them
* If not, then lets unquote all of them